frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -65,7 +67,7 @@ const TableCell = styled(MuiTableCell)({
   fontSize: "14px",
   fontWeight: "normal",
   height: "48px",
-  padding: "0 16px",
+  padding: "15px 16px 15px 16px",
```

Comment:
based on figma design for table cell

Reply:
This one is hard because it depends on which cell you are looking at. Some of them have top & bottom padding but others don't. I think for now we shouldn't have any and just center the items vertically

Reply:
okay sounds good. i reverted to what we currently have (`padding: "0 16px"`) bc didn't look good w/o some right padding 

ex w/o the right padding
<img width="600" alt="Screen Shot 2020-12-16 at 5 52 15 PM" src="https://user-images.githubusercontent.com/39421794/102421064-82329580-3fd1-11eb-8c68-f3664e6ff795.png">